app/models/movie.rb

```diff
@@ -1,5 +1,7 @@
 class Movie < ApplicationRecord
-  belongs_to :genre
-  has_and_belongs_to_many :clients
-  belongs_to :cinema_hall
+  belongs_to :genre, foreign_key: 'genre_id'
```

Comment:
```suggestion
  belongs_to :genre
```
It's by default, don't overcomplicate it, please.